ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -1565,16 +1619,26 @@ def test__execute_cleaning_foreign_drives(self,
             'is_reboot_required': constants.RebootRequired.false,
             'is_commit_required': False
         }
+        mock_commit_config.side_effect = ['42']
+        mock_change_physical_disk_state.return_value = {
+            'is_reboot_required': True,
+            'conversion_results': {
+            'RAID.Integrated.1-1': {'is_reboot_required': 'optional',
+                                    'commit_required': False,
+                                    'is_commit_required': True}},
+            'commit_required_ids': ['RAID.Integrated.1-1']}
 
         with task_manager.acquire(self.context, self.node.uuid,
                                   shared=False) as task:
             return_value = task.driver.raid._execute_cleaning_foreign_drives(
                 task, self.node)
-            mock_resume.assert_called_once_with(
-                task, 'cleaning', 'continue_node_clean')
+            #mock_resume.assert_called_once_with(
+            #    task, 'cleaning', 'continue_node_clean')
 
         self.assertIsNone(return_value)
-        self.assertNotIn('raid_config_parameters',
-                         self.node.driver_internal_info)
-        self.assertNotIn('raid_config_substep',
-                         self.node.driver_internal_info)
+        self.assertEqual(['42'],
+            self.node.driver_internal_info['raid_config_job_ids'])
+        #self.assertNotIn('raid_config_parameters',
+        #                 self.node.driver_internal_info)
```

Comment:
Should remove these commented out asserts and add asserts on raid_config-substep and raid_config_parameters.

Reply:
removed commented asserts and added asserts required.